tests/Browser/LoginTest.php

```diff
@@ -0,0 +1,231 @@
+<?php
+
+namespace Tests\Browser;
+
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Laravel\Dusk\Browser;
+use Tests\DuskTestCase;
+use App\User;
+use Hash;
+
+class LoginTest extends DuskTestCase
+{
+    /**
+     * A basic browser url not login.
+     */
+    public function testPathWrong()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/abc')
+                    ->assertSee('404');
+        });
+    }
+
+    /**
+     * A basic browser go to path admin dislay Login.
+     */
+    public function testPathAdmin()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->visit(config('contants.path.admin'))
+                    ->assertPathIs(config('contants.path.login'));
+        });
+    }
+
+    /**
```

Comment:
Please match and reorder test case follow order in Test Case Report

Reply:
I changed function to match Test case report.